src/public/pages/log_viewer/index.css

```
@@ -0,0 +1,376 @@
+}
+.log-level-debug {
+	opacity: 0.6;
+}
+.log-row.log-level-log,
+.log-row.log-level-stdout {
+	border-left: 3px solid transparent;
+}
+.log-content {
+	flex: 1;
+	min-width: 0;
+	word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)